app/src/main/java/com/example/codingquiz/ui/screen/QuestionScreen.kt

```diff
@@ -47,11 +50,12 @@ class AnswerState {
 fun QuestionScreen(
     questionViewModel: QuestionViewModel = koinViewModel(),
     givenAnswerViewModel: GivenAnswerViewModel = koinViewModel(),
-    categoryId: Int?,
+    category: Category?,
     onBackPressed: (List<QuizResult>) -> Unit,
     navigateToResults: (List<QuizResult>) -> Unit,
 ) {
     val question by questionViewModel.question.collectAsState()
```

Comment:
```suggestion
    val question by questionViewModel.question.collectAsStateWithLifecycle()
```

Reply:
Everytime you use collectAsState I think

Reply:
Thanks for suggestion :)